src/connector/src/sink/iceberg/mod.rs

```diff
@@ -1103,9 +1148,10 @@ impl IcebergSinkWriterInner {
             )
             .build();
 
-        let parquet_writer_builder = ParquetWriterBuilder::new(
-            parquet_writer_properties,
-            schema.clone(),
+        let parquet_writer_builder =
+            ParquetWriterBuilder::new(parquet_writer_properties, schema.clone());
+        let rolling_builder = RollingFileWriterBuilder::new_with_default_file_size(
```

Comment:
Is using **default_file_size** the expected behavior?

Reply:
Yes. Currently, we don't have such a parameter to control the target file size for writer, but we have a similar parameter for iceberg compaction.